packages/vaadin-virtualizer/src/iron-list.js

```diff
@@ -1,44 +1,27 @@
 /**
- @license
```

Comment:
This file is trimmed copy of https://github.com/PolymerElements/iron-list/blob/master/iron-list.js